opengrok-indexer/src/main/java/org/opengrok/indexer/index/DefaultIndexChangedListener.java

```diff
@@ -41,21 +41,25 @@
 
     @Override
     public void fileAdd(String path, String analyzer) {
-        LOGGER.log(Level.INFO, "Add: {0} ({1})", new Object[]{path, analyzer});
+        if (LOGGER.isLoggable(Level.FINE)) {
```

Comment:
why do you use if condition here and not in fileRemove or fileUpdate?

Reply:
To short-circuit `Object` creation happening before the `log()` call, which the others are not doing

Reply:
ah, ok.